docker/Dockerfile

```diff
@@ -1,14 +1,23 @@
 FROM ubuntu:16.04
 MAINTAINER prozlach@mesosphere.io
 
-ENV OPENRESTY_VERSION 1.9.15.1
-ENV OPENRESTY_DOWNLOAD_URL https://openresty.org/download/openresty-$OPENRESTY_VERSION.tar.gz
-ENV OPENRESTY_DOWNLOAD_SHASUM 491a84d70ed10b79abb9d1a7496ee57a9244350b
-
-ENV VENV_DIR /usr/local/venv
-ENV AR_CONF_DIR /usr/local/src/adminrouter
-ENV OPENRESTY_DIR /usr/local/src/openresty
-ENV AR_BIN_DIR /usr/local/adminrouter
+# Let's try to limit the number of layers to minimum:
+ENV OPENRESTY_VERSION=1.9.15.1 \
+    OPENRESTY_DOWNLOAD_SHASUM=491a84d70ed10b79abb9d1a7496ee57a9244350b \
+    OPENRESTY_DIR=/usr/local/src/openresty \
+    OPENRESTY_COMPILE_SCRIPT=/usr/local/src/build-resty.sh \
+    OPENRESTY_COMPILE_OPTS="" \
+    VENV_DIR=/usr/local/venv \
+    AR_BIN_DIR=/usr/local/adminrouter/ \
+    VEGETA_DOWNLOAD_SHA256=2f0a69d0ae6f0bf268b7f655bd37c0104d5568d5b2bc45bbb2c405266f74e33d \
+    VEGETA_DOWNLOAD_URL=https://github.com/tsenart/vegeta/releases/download/v6.1.1/vegeta-v6.1.1-linux-amd64.tar.gz \
+    IAM_PUBKEY_FILE_PATH=/usr/local/iam.jwt-key.pub \
+    IAM_PRIVKEY_FILE_PATH=/usr/local/iam.jwt-key.priv \
+    IAM_SHARED_SECRET_FILE_PATH=/usr/local/iam.jwt-key.shared-secret
+
+# These depend on other ENV vars, so we need a separate ENV block:
+ENV OPENRESTY_DOWNLOAD_URL=https://openresty.org/download/openresty-$OPENRESTY_VERSION.tar.gz \
```

Comment:
We are shipping OpenResty sources inside devkit container instead of "buidinfo.json/ee.buildinfo.json". This may be already confusing for some people and moving version to Makefile and to DC/OS'es 'build' script may confuse people even more ("Where is it?"), not to mention that we will have two places where it is defined and that need to be kept in sync.